fonts.css

```diff
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: 'Montserrat';
+  src: 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)